functions/handlers/rooms.js

```diff
@@ -3,4 +3,26 @@ const admin = require('firebase-admin');
 
 exports.close = ({ params: { roomId } }) => admin.database().ref('/messages').child(roomId).remove();
 
-exports.empty = ({ params: { roomId } }) => admin.database().ref('/rooms').child(roomId).remove();
\ No newline at end of file
+exports.empty = ({ params: { roomId } }) => admin.database().ref('/rooms').child(roomId).remove();
+
+exports.create = ({ params: { roomId } }) => {
+  const roomName = admin.database().ref('rooms').child(roomId).child('name');
+
+  return admin.messaging().sendToTopic('/topics/public', {
```

Comment:
mozna by wrzucic `co` i generatorem to pocisnąć zamiast robić takie łączenie Promise'ów ale już nie ma czasu na to :)